phpBB/styles/prosilver/theme/common.css

```diff
@@ -3,8 +3,6 @@
 /* -------------------------------------------------------------- */
 /* stylelint-disable selector-max-compound-selectors */
 /* stylelint-disable selector-no-qualifying-type */
-/* stylelint-disable declaration-property-unit-whitelist */
-/* stylelint-disable declaration-property-unit-blacklist */
```

Comment:
you are not or have not achieved a proper re-base as this should not be removed. The issue is that you are trying to rebase and then apply changes that you made from a 3.2 or older master version of these files which is causing conflicts. all the css changes will need to be reverted and then manually re-added to properly clean things up.